.storybook-v3/custom-addons/provider/index.js

```diff
@@ -16,7 +16,7 @@ function ProviderUpdater(props) {
   let [localeValue, setLocale] = useState(providerValuesFromUrl.locale || undefined);
   let [themeValue, setTheme] = useState(providerValuesFromUrl.theme || undefined);
   let [scaleValue, setScale] = useState(providerValuesFromUrl.scale || undefined);
-  let [toastPositionValue, setToastPosition] = useState(providerValuesFromUrl.toastPosition || 'top');
+  let [toastPositionValue, setToastPosition] = useState(providerValuesFromUrl.toastPosition || 'bottom');
```

Comment:
Just curious, why the change from having toast position at the bottom of the storybook instead of at the top?

Reply:
spectrum design docs say toasts should be on the bottom